src/main/java/net/minecraftforge/network/PlayMessages.java

```diff
@@ -142,7 +142,7 @@ public static void handle(SpawnEntity msg, Supplier<NetworkEvent.Context> ctx)
                     return;
                 }
 
-                e.setPos(msg.posX, msg.posY, msg.posZ);
+                e.syncPacketPositionCodec(msg.posX, msg.posY, msg.posZ);
```

Comment:
Could you please add a comment above this to indicate that this mirrors what `Entity#recreateFromPacket`/`LivingEntity#recreateFromPacket` does, for future convenience by readers and patchers?